mantidproject/mantid
# FlatPlatePaalmanPings input fixes

Changes:
1. Interface `Indirect Corrections -> Calculate Paalman Pings -> Container Back Thickness` has been fixed to set the container back thickness correctly.
1. The algorithm should now deal with cases of no sample, no can back thickness and no can front thickness (and any combinations of these) without generaing `NaNs`. A warning is logged in these cases.

**To test:**

Look at updated system test. Try running these manually to verify that the output workspaces are free of `NaNs`. In the case of no sample and no can the workspace should be all `1`s.

Fixes #21520 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
